gulp-tasks/workbox:generate-contributors.js

```diff
@@ -0,0 +1,94 @@
+const GitHubApi = require('github');
+const fs = require('fs-extra');
+const path = require('path');
+const gulp = require('gulp');
+
+const MAX_COLLABORATORS = 20;
+
+function getContributorHTML(contributorStats) {
+  const contributor = contributorStats.author;
+  var html = `
+<div class="wb-contrib">
```

Comment:
can we do this using templates so we can keep code to build and template code separate?

Reply:
Can you give me an example of doing this please - I struggle with figuring out the devsite stuff at the best of times.